### PR TITLE
feat: [ADLN] UPD config update

### PR DESCRIPTION
**Pull request by tsaikevin**
Aligned FSPS UPD settings with BIOS

Signed-off-by: Kevin Tsai <kevin.tsai@intel.com>

### CODE REVIEW

File: Platform/AlderlakeBoardPkg/Library/FspsUpdUpdateLib/FspsUpdUpdateLib.c [hunk 1]
```diff
@@ -1128,10 +1128,10 @@ UpdateFspConfig (
       case PLATFORM_ID_ADL_N_DDR5_CRB:
         FspsConfig->PchLanEnable = 0x0;
         FspsConfig->PchPmVrAlert = 0x1;
-        FspsConfig->PchFivrExtV1p05RailEnabledStates = 0x3f;
+        FspsConfig->PchFivrExtV1p05RailEnabledStates = 0x1f;
         FspsConfig->PchFivrExtVnnRailEnabledStates = 0x3f;
         FspsConfig->PchFivrExtVnnRailSupportedVoltageStates = 0x4;
-        FspsConfig->PchFivrVccinAuxOffToHighCurModeVolTranTime = 0x0;
+        FspsConfig->PchFivrVccinAuxOffToHighCurModeVolTranTime = 0x96;
```
Comment (sindhuragrandhi) [inline Platform/AlderlakeBoardPkg/Library/FspsUpdUpdateLib/FspsUpdUpdateLib.c:1134]: @kvparikh89 seems like BIOS is still setting this to 150 by default. I remember they are supposed to revert to zero for PV release. Do you recall?

Comment (sindhuragrandhi) [reply Platform/AlderlakeBoardPkg/Library/FspsUpdUpdateLib/FspsUpdUpdateLib.c:1134]: Got some inputs from BIOS. This issue is resolved. 0x96 is good as a default.

**Review by sindhuragrandhi — APPROVED** [review]

**Review by kvparikh89 — APPROVED** [review]
Looks  good to me

File: Platform/AlderlakeBoardPkg/Library/FspsUpdUpdateLib/FspsUpdUpdateLib.c (hunk 1 above)
Comment (kvparikh89) [reply Platform/AlderlakeBoardPkg/Library/FspsUpdUpdateLib/FspsUpdUpdateLib.c:1134]: Thanks for clarifying

**Review by gdong1 — APPROVED** [review]

**Pull request merged**